IdentityCore/src/controllers/broker/mac/MSIDXpcSilentTokenRequestController.m

```diff
@@ -55,4 +79,24 @@ + (BOOL)canPerformRequest
     }
 }
 
+- (void)shouldSkipAcquireTokenBasedOn:(NSError *)error
+{
+    self.shouldSkipAcquireToken = YES;
```

Comment:
What if error is nil? This method will log incorrect info which might confuse in case of issue debugging